### PR TITLE
Add company-proximity-navigator sample

### PR DESCRIPTION
**Pull request by atanasiuk-hubspot**

### CODE REVIEW

File: company-proximity-navigator/src/app/extensions/dist/NearestCompanies-manifest.json [hunk 1]
```diff
@@ -0,0 +1,8856 @@
+{
```
Comment (theJoestJoeToEverJoe) [inline company-proximity-navigator/src/app/extensions/dist/NearestCompanies-manifest.json:1]: The contents of the `dist` directory can be deleted and added to a `.gitignore`

File: company-proximity-navigator/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (theJoestJoeToEverJoe) [inline company-proximity-navigator/package-lock.json:1]: Can you remove all the package-lock.json files in the project and add them to a `.gitignore` please?

File: company-proximity-navigator/src/app/extensions/TopValueCompanies.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      {topValueCompaniesSorted.length > 0 && (
+        <CompaniesWithDistanceTable
+          portalId={context.portal.id}
+          companies={topValueCompaniesSorted.slice(0, companiesToDisplay)}
+          propertiesToDisplay={[
+            { title: 'Annual Revenue', propertyName: 'annualrevenue' },
+          ]}
+        />
+      )}
+      <ButtonRow>
```
Comment (theJoestJoeToEverJoe) [inline company-proximity-navigator/src/app/extensions/TopValueCompanies.jsx:76]: Do we need this button row?

**Review by theJoestJoeToEverJoe — COMMENTED** [review]
LGTM just a few minor comments

File: company-proximity-navigator/src/app/extensions/TopValueCompanies.jsx (hunk 3 above)
Comment (atanasiuk-hubspot) [reply company-proximity-navigator/src/app/extensions/TopValueCompanies.jsx:76]: Similar to https://github.com/HubSpot/ui-extensions-examples/pull/30#discussion_r1305369651

Comment (atanasiuk-hubspot) [reply company-proximity-navigator/src/app/extensions/TopValueCompanies.jsx]: Replaced with `Box`

**Review by Randyjp — APPROVED** [review]

**Pull request merged**